.gitignore

```diff
@@ -1,4 +1,4 @@
 /.idea/
 /.gradle
 /local.properties
-/build/*
+**/build
```

Comment:
The JS target adds a bunch of files in jsApp/build. Should this be in a .gitignore in jsApp?